packages/components/pages/f-mfa/src/components/Mfa.vue

```diff
@@ -43,11 +153,41 @@ export default {
 .c-mfa {
     display: flex;
     justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
     margin: auto;
-    border: 1px solid red;
-    font-family: f.$font-family-base;
-    @include f.font-size(heading-m);
+    text-align: center;
+
+    @include f.media('>=narrow') {
+        width: 80vw;
```

Comment:
Should there be a `max-width` set for this for users on very wide screens? Otherwise, I imagine it could get a bit wide at very large widths